### PR TITLE
Add Footable sort, pagination and filter

### PR DESCRIPTION
**Pull request by vidya-ram**
Add footable sort, pagination and filter for boxoffice admin orders page [https://github.com/hasgeek/boxoffice/pull/57]

### CODE REVIEW

File: baseframe/assets.py [hunk 1]
```diff
@@ -148,7 +148,10 @@
 
 # FooTable
 assets['footable.js'][Version('2.0.3')] = ('jquery.js', 'baseframe/js/footable.js')
-assets['footable.css'][Version('2.0.3')] = 'baseframe/css/footable.css'
+assets['footable-paginate.js'][Version('2.0.3')] = ('jquery.js', 'footable.js', 'baseframe/js/footable-paginate.js')
+assets['footable-filter.js'][Version('2.0.3')] = ('jquery.js', 'footable.js', 'baseframe/js/footable-filter.js')
+assets['footable-sort.js'][Version('2.0.3')] = ('jquery.js', 'footable.js', 'baseframe/js/footable-sort.js')
+assets['footable.css'][Version('2.0.3')] = ('fontawesome.css', 'baseframe/css/footable.css')
```
Comment (jace) [inline baseframe/assets.py]: Does footable depend on FontAwesome? What version?

**Comment by jace** [comment]
@vidya-ram if you're making extensive customisations to footable.css, don't give it the same name. Make it `footable-baseframe.css` (or `baseframe-footable.css`, check which convention we're using) and in assets declare that it provides `footable.css`.

File: baseframe/assets.py (hunk 1 above)
Comment (vidya-ram) [inline baseframe/assets.py]: I customized `footable.css` to use `fontawesome` icons.

Comment (vidya-ram) [inline baseframe/assets.py]: Fixed.

File: baseframe/assets.py [hunk 2]
```diff
@@ -148,7 +148,10 @@
 
 # FooTable
 assets['footable.js'][Version('2.0.3')] = ('jquery.js', 'baseframe/js/footable.js')
-assets['footable.css'][Version('2.0.3')] = 'baseframe/css/footable.css'
+assets['footable-paginate.js'][Version('2.0.3')] = ('jquery.js', 'footable.js', 'baseframe/js/footable-paginate.js')
+assets['footable-filter.js'][Version('2.0.3')] = ('jquery.js', 'footable.js', 'baseframe/js/footable-filter.js')
+assets['footable-sort.js'][Version('2.0.3')] = ('jquery.js', 'footable.js', 'baseframe/js/footable-sort.js')
+assets['footable.css'][Version('2.0.3')] = ('fontawesome.css==4.4.0', 'baseframe/css/baseframe-footable.css')
```
Comment (jace) [inline baseframe/assets.py]: No, not this. You need this:

```python
assets['baseframe-footable.css'][Version('2.0.3')] = {
    'provides': 'footable.css',
    'requires': 'fontawesome.css>=4.4.0',
    'bundle': 'baseframe/css/baseframe-footable.css',
    }
```

**Pull request merged**